imagebuildah/chroot_symlink.go

```diff
@@ -145,3 +239,11 @@ func hasSymlink(path string) (bool, string, error) {
 	// run filepath.Clean to remove the ".." from relative paths
 	return true, filepath.Clean(targetDir), nil
 }
+
+// boolToString converts a boolean to its equivalent string form
```

Comment:
Is there a reason to not use `fmt.*rintf("%v", bool)`?

Reply:
Nope, just didn't think about it. Thanks for the tip! :)